src/applications/vaos/tests/new-appointment/components/ConfirmationPage/ConfirmationRequestInfo.unit.spec.jsx

```diff
@@ -189,30 +197,39 @@ describe('VAOS <ConfirmationRequestInfo>', () => {
     };
     const pageTitle = 'Your appointment request has been submitted';
 
-    const tree = mount(
+    const screen = renderWithStoreAndRouter(
       <ConfirmationRequestInfo
         data={data}
         facilityDetails={facilityDetails}
         pageTitle={pageTitle}
       />,
+      {
+        initialState,
+        reducers,
```

Comment:
You can leave out reducers here and in other spots, it's included by default in `renderWithStoreAndRouter`.

Reply:
Done.